src/exampleTypes/ResultTable.js

```diff
@@ -116,6 +116,18 @@ let Header = withStateLens({ popover: false, adding: false, filtering: false })(
       tree.mutate(filterNode.path, { paused: false })
       F.flip(filtering)()
     }
+    const moveColumn = (computeNewIndex, field, visibleFields) => {
+      let visibleFieldIndex = _.findIndex(x => x.field === field, visibleFields)
+      let nextField = _.flow(
+        _.nth(computeNewIndex(visibleFieldIndex)),
+        _.get('field')
+      )(visibleFields)
+      let currentIndex = _.findIndex(x => x === field, [...includes])
+      let nextFieldIndex = _.findIndex(x => x === nextField, [...includes])
```

Comment:
Since you're matching exact values, this would be cleaner with `_.indexOf`:

```js
let currentIndex = _.indexOf(field, [...includes])
let nextFieldIndex = _.indexOf(nextField, [...includes])
```

Reply:
I'm also not sure we actually need to spread

Reply:
Fixed